webapp/src/ts/services/android-api.service.ts

```diff
@@ -241,6 +243,17 @@ export class AndroidApiService {
     this.geolocationService.permissionRequestResolved();
   }
 
+  resolveCHTExternalAppResponse(response) {
+    try {
+      this.externalAppLauncherService.resolveExternalAppResponse(response);
+    } catch (error) {
+      console.error(
+        `AndroidApiService :: Error when processing response from external app, error: "${error.message}, response:"`,
+        response
+      );
+    }
```

Comment:
Looking at the code I can't imagine how this could ever throw an error. Any idea, or are you just being extra careful here?

Reply:
I think too extra defensive lol, I've removed it.